### ISSUE TITLE
Unable to pair by QR Code and Sign In methods

### ISSUE THREAD

**Issue by kw0n** [OrangeDrangon/android-messages-desktop#449]
**Describe the bug**
I have been unable to pair the app with my phone as usual through QR Code and/or Sign In methods (both top and bottom sign in buttons). When trying to pair with QR code, The QR scanner from the Messages app on my phone reads and error stating "Something went wrong. Try again in a few minutes."
When trying to pair by the Sign In buttons on the top right and the bottom.

**To Reproduce**
1. Open 'Google Messages app v5.4.2 on Windows 11' and Google Messages app in android phone (S23)
2. On the S23, Click on 'Profile picture then Device Pairing then use with and/or without an account. '
3a. When using without an account, click 'Switch to QR pairing' then 'QR code scanner' then scan the QR Code within the Windows Google Messages v.5.4.2 app. From here it will display an error labeled under 4a.
3b. When using with an account, it will display error in 4b.
4a. See error - "Something went wrong. Try again in a few minutes."
4b. See error - "Can't pair with your Google Account right now. Restart the Messages app and sign in again."

**Expected behavior**
Work as usual

**Screenshots**
![1](https://github.com/OrangeDrangon/android-messages-desktop/assets/13937442/235507b0-cdf3-4235-96cd-fabc72980aeb)
![2](https://github.com/OrangeDrangon/android-messages-desktop/assets/13937442/17078746-cd27-4b06-b93d-618bf7ea7fc2)
![3](https://github.com/OrangeDrangon/android-messages-desktop/assets/13937442/349d8a58-c577-4161-9ed5-951eba743d12)

**Desktop (please complete the following information):**
 - OS: [Windows 11 Version 23H2 (OS Build 22631.3593)]
 - OS: Android 14 - Samsung OneUI (Using Samsung Galaxy S23)

**Additional context**
This only started happening around 05/27/2024, prior to this it was working without issue.

**Comment by OrangeDrangon** [comment]
Likely will be fixed in latest release

**Comment by kw0n** [comment]
just reporting back, I wasn't sure if you meant the release from today 06/08/2024 but it still has the issues above

**Comment by hisnameisjoel** [comment]
Just wanted to report that I was able to sign in with no issues using the Google Account + Messages verify method (where it has you select the right emoji) which obviously means the old QR code scanning method is out. This morning I updated the Messages app on my Galaxy S24 Ultra, so it's possible that @kw0n you'll need to update the app on your phone to enable the new device pairing method.

**Comment by kw0n** [comment]
Hey Joel, thanks for letting me know. I went back in to troubleshoot. I deleted the appdata messages on PC, uninstalled messages updates on s23 and reinstalled the latest version of google messages.  From here it still didnt work for some reason, then I changed dns on my pc and it allowed me to connect. I then changed back to the prior dns I had it set to and it still worked for both QR and sign in modes. I restarted both PC and S23 and it still is working, allowing me to pair and unpair at will. Thank you @OrangeDrangon for keeping this alive and updating when needed and thanks @hisnameisjoel for giving insight.
